wpilib/wpilib/sensorbase.py

```diff
@@ -68,7 +69,8 @@ def checkSolenoidModule(moduleNumber):
         :param moduleNumber: The solenoid module module number to check.
         """
         if not hal.checkSolenoidModule(moduleNumber):
-            raise IndexError("Requested solenoid module number %d is out of range [0, %d)." % (moduleNumber, SensorBase.kPCMModules))
```

Comment:
Cmon, if you're going to do a formatting PR, do it separately. This makes reviewing really annoying.

Reply:
Not to mention that the reformatting feels really awkward here.